src/tokens/lst_tokens/token_lst_del_one.c

```diff
@@ -36,5 +36,5 @@ void	token_lst_del_one(t_token_lst *const list, t_token *const node)
 	}
 	--list->size;
 	free(node->str);
-	free(node);
+	free((void *)node);
```

Comment:
Not convinced.

Reply:
After some investigations, I believe that it is a better idea to keep it const and cast it as non-const only when calling `free()` (cf https://stackoverflow.com/questions/2819535/unable-to-free-const-pointers-in-c#comment2860291_2819594)